win32ss/reactx/ntddraw/dxeng.c

```diff
@@ -465,7 +465,11 @@ DxEngGetDCState(HDC hDC,
                 break;
             case 2:
```

Comment:
Unrelated to this specific PR, but an idea for thought, it would be nice to have explicit names for this type values.

Reply:
I wonder is it possible at all to see the aliases (names) of these types via debugging analysis somehow... :thinking:

Reply:
I don't think so.